packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -138,15 +138,17 @@ library ChainAssertions {
             require(address(messenger.otherMessenger()) == Predeploys.L2_CROSS_DOMAIN_MESSENGER, "CHECK-L1XDM-30");
             require(address(messenger.PORTAL()) == _contracts.OptimismPortal, "CHECK-L1XDM-40");
             require(address(messenger.portal()) == _contracts.OptimismPortal, "CHECK-L1XDM-50");
-            require(address(messenger.superchainConfig()) == _contracts.SuperchainConfig, "CHECK-L1XDM-60");
+            require(
```

Comment:
Let's replace with the system config check instead of checking superchain config